### PR TITLE
Ledger + Lunie guide

### PR DESCRIPTION
**Pull request by gamarin2**
An improvement to the existing ledger guide with screenshots

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: docs/ledger-lunie-guide.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Using Lunie with the Ledger Nano S
```
Comment (alexanderbez) [inline docs/ledger-lunie-guide.md]: What's the point of this file? It should at least link to something...

**Review by alexanderbez — APPROVED** [review]
ACK

File: docs/ledger-lunie-guide.md (hunk 1 above)
Comment (fedekunze) [reply docs/ledger-lunie-guide.md]: ++. Please remove the file

File: docs/ledger.md [hunk 2]
```diff
@@ -8,13 +8,63 @@ At the core of a Ledger device there is a mnemonic seed phrase that is used to g
...
+2. Using Ledger Live, [update your Ledger Nano S with the latest firmware](https://support.ledger.com/hc/en-us/articles/360002731113-Update-device-firmware).
+3. On the Ledger Live application, navigate to the `Manager` menu . 
+    ![manager](./ledger-tuto-manager.png)
+4. Connect your Ledger Nano device and allow Ledger Manager from it. 
+5. On the Ledger Live application, Search for `Cosmos`. 
+    ![search](./ledger-tuto-search.png)
+6. Install the Cosmos application by clicking on `Install`.
+
+::: tip
+To see the `Cosmos` application when you search for it, you might need to activate the `Developer Mode`, located in the Experimental features tab of the Ledger Live application.
```
Comment (fedekunze) [inline docs/ledger.md:25]:
```suggestion
To see the `Cosmos` application when you search for it, you might need to activate the `Developer Mode`, located in the Experimental features tab of the Ledger Live application.
:::
```

File: docs/ledger.md [hunk 3]
```diff
@@ -8,13 +8,63 @@ At the core of a Ledger device there is a mnemonic seed phrase that is used to g
...
+
+![Devmode](./ledger-tuto-dev-mode.png)
+
+## Lunie.io + Ledger Nano  
+
+**Note: You need to [install the Cosmos app](#install-the-cosmos-ledger-application) on your Ledger Nano before using following this section**
+
+1. Connect your Ledger device to your computer, unlock it with the PIN and open the Cosmos app.
+2. Open [https://lunie.io](https://app.lunie.io/existing) in your web browser (latest version of Google Chrome preferred).
```
Comment (fedekunze) [inline docs/ledger.md]: 😃 
```suggestion
2. Open [https://app.lunie.io](https://app.lunie.io) in your web browser (latest version of [Brave](https://brave.com/) preferred).
```

File: docs/ledger.md [hunk 4]
```diff
@@ -8,13 +8,63 @@ At the core of a Ledger device there is a mnemonic seed phrase that is used to g
...
+3. Choose `Sign in with Ledger Nano S`.
```
Comment (fedekunze) [inline docs/ledger.md:36]:
```suggestion
3. Choose `Use an existing address` and then  [`Use Ledger Nano`](https://app.lunie.io/ledger) as sign-in option.
```

File: docs/ledger.md [hunk 5]
```diff
@@ -8,13 +8,63 @@ At the core of a Ledger device there is a mnemonic seed phrase that is used to g
...
+    ![lunie-option](./ledger-tuto-lunie-option.png)
+4. Click on `Sign in`. Do not forget to open the Cosmos app on your Ledger device for Lunie to recognize it.
```
Comment (fedekunze) [inline docs/ledger.md]:
```suggestion
4. Make sure your Ledger device is unlocked and with the Cosmos app open and then click on the`Sign in` button. 
```

File: docs/ledger.md [hunk 6]
```diff
@@ -8,13 +8,63 @@ At the core of a Ledger device there is a mnemonic seed phrase that is used to g
...
+    ![lunie-option](./ledger-tuto-lunie-option.png)
+4. Click on `Sign in`. Do not forget to open the Cosmos app on your Ledger device for Lunie to recognize it. 
+
+That's it! You can now use Lunie with your Ledger Nano S. You will find your Cosmos address here:
+
+![lunie-address](./ledger-tuto-lunie-address.png)
+
+**Note: Each time you will send a transaction, you will need to confirm it on your Ledger device. Indication will be prompted from the Lunie interface**
+
+### (Optional) Confirm your address
```
Comment (fedekunze) [inline docs/ledger.md:46]: you can do this directly on the Cosmos Ledger app without relying on the CLI. Please add that instead

**Review by fedekunze — CHANGES_REQUESTED** [review]
Thanks @gamarin2. Minor changes requested

File: docs/ledger-lunie-guide.md (hunk 1 above)
Comment (gamarin2) [reply docs/ledger-lunie-guide.md]: no point, I just forgot to delete it

File: docs/ledger.md (hunk 4 above)
Comment (gamarin2) [reply docs/ledger.md:36]: The link I provide above is consistent with the recommendation

**Comment by gamarin2** [comment]
Can we merge @alexanderbez

**Review by fedekunze — APPROVED** [review]

**Pull request merged**